app/models/setting.rb

```diff
@@ -100,11 +101,13 @@ def self.method_missing(method, *args)
 
   def value=(v)
     v = v.to_yaml unless v.nil?
+    v = encrypt_field(v) if defined? encrypted and encrypted
```

Comment:
Prefer `has_attribute?(:encrypted)` perhaps?  A comment to say why you're performing this check would be useful - I'm assuming it's to enable DB migrations on older versions?